src/style.css

```diff
@@ -1,3 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 12px;
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  width: 100%;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.img__logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: block;
+  color: black;
```

Comment:
Ensure that the color contrast between the text and background is sufficient for readability, especially for users with visual impairments. Consider using a tool to check color contrast ratios.